syntax = "proto3";

package protos;

option go_package = "github.com/wwqdrh/cloard/runtime/protos";

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}